src/NServiceBus.Transport.RabbitMQ.Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -55,6 +55,10 @@ namespace NServiceBus
             "rt.MessageIdStrategy` instead. The member currently throws a NotImplementedExcep" +
             "tion. Will be removed in version 8.0.0.", true)]
         public static NServiceBus.TransportExtensions<NServiceBus.RabbitMQTransport> CustomMessageIdStrategy(this NServiceBus.TransportExtensions<NServiceBus.RabbitMQTransport> transportExtensions, System.Func<RabbitMQ.Client.Events.BasicDeliverEventArgs, string> customIdStrategy) { }
+        [System.Obsolete("The TimeoutManager has been removed from NServiceBus 8. See the upgrade guide for" +
+            " details on how to use the timeout migration tool. The member currently throws a" +
+            " NotImplementedException. Will be removed in version 8.0.0.", true)]
+        public static void DelayedDelivery(this NServiceBus.TransportExtensions<NServiceBus.RabbitMQTransport> transportExtensions) { }
```

Comment:
do we need to return the original `DelayedDeliverySettings` type and obsolete that one as well?

Reply:
I was too lazy to do it. I think in this case it is not needed as there is going to be compilation error caused by the usage of an obsoleted member. There is also going to be an error caused by the type mismatch but I would call yagni on this.